src/onzr/tui.py

```diff
@@ -0,0 +1,133 @@
+"""Onzr: TUI elements."""
+
+from textual.app import App, ComposeResult
+from textual.containers import Container, HorizontalGroup, Vertical
+from textual.reactive import reactive
+from textual.widgets import (
+    Button,
+    DataTable,
+    Footer,
+    Header,
+    Static,
+    TabbedContent,
+    TabPane,
+)
+
+from onzr import cli
+from onzr.client import OnzrClient
+from onzr.models import PlayingState, QueuedTracks
+
+
+def get_now_playing():
+    """Get the current playing status."""
+    client = OnzrClient()
+    playing_state: PlayingState = client.now_playing()
+    if playing_state.track is not None:
+        return playing_state.track.title
+    else:
+        return "Nothing"
```

Comment:
What about getting onzr API client instead? Cache this client and attach it to the App "state"?


```suggestion
@cached
def get_onzr_client():
    """Get the onzr client."""
    return OnzrClient()
```